Kernel/CMakeLists.txt

```diff
@@ -75,14 +75,14 @@ set(KERNEL_SOURCES
     Graphics/DisplayConnector.cpp
     Graphics/Generic/DisplayConnector.cpp
     Graphics/GraphicsManagement.cpp
+    Graphics/Intel/Auxiliary/DisplayPortAuxiliaryConnector.cpp
+    Graphics/Intel/Auxiliary/GMBusConnector.cpp
```

Comment:
All of the files moved in `Kernel/IntelGraphics: Move auxiliary connectors to a separate directory `  are new to this PR, so let's put them there in the initial commit.